### DISCUSSION TITLE
Trace operation '/*' in spring-boot application

### DISCUSSION THREAD

**Discussion by bgong-mdsol** [open-telemetry/opentelemetry-java-instrumentation#4653] [Q&A]
Hello.
Our spring app is running with opentelemetry-javaagent version 1.6.0-SNAPSHOT, we notice there are some traces which operation is "/*" and its http status is 200 or 304, I'm not sure what exactly the span name is set, can someone give a hint?
Thanks in advance

**Comment by trask** [comment]
> Hello. Our spring app is running with opentelemetry-javaagent version 1.6.0-SNAPSHOT, we notice there are some traces which operation is "/*" and its http status is 200 or 304, I'm not sure what exactly the span name is set, can someone give a hint? Thanks in advance

hi @bgong-mdsol! what's the `http.url` or `http.target` attribute on the `SERVER` span, and what does that map to in your code? e.g. spring controller or something else?

**Reply by bgong-mdsol** [reply]
@trask Here is a RestController class we got the trace which operation is "/*"
```
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;
import org.springframework.web.bind.annotation.RequestMapping;
import lombok.RequiredArgsConstructor;
...

@RestController("FilesV2")
@RequestMapping(value = { "/apps/api/v2/files" })
@RequiredArgsConstructor
public class FileController {
   ...
   @GetMapping(value = "/{id}", produces = JSON)
   ...
}
```

`http.url`: {{BaseHostUrl}}`/apps/api/v2/files/1?pageSize=15`
`http.status_code`: 304
`cloud.provider`: aws

**Reply by trask** [reply]
@bgong-mdsol is the 304 coming from the controller? or is there something in front of the controller (e.g. filter) that is returning the 304?

**Reply by bgong-mdsol** [reply]
@trask it comes from the filter which extends from org.springframework.web.filter.OncePerRequestFilter

**Reply by iNikem** [reply]
@bgong-mdsol If your HTTP request is served already by a filter and never actually reaches your MVC controller, then MVC controller instrumentation does not kick in and has no chance to update server span name
